process/datadog_checks/process/process.py

```diff
@@ -213,7 +213,7 @@ def find_pids(self, name, search_string, exact_match, ignore_ad=True):
             self.last_ad_cache_ts[name] = time.time()
         return matching_pids
 
-    def psutil_wrapper(self, process, method, accessors, try_sudo, *args, **kwargs):
+    def psutil_wrapper(self, process, method, accessors=None, *args, **kwargs):
```

Comment:
I think the change to the function signature makes a ton of sense, but I would rather do it on a separate PR to not change two things at once here.